src/pybind/mgr/dashboard/frontend/src/app/shared/directives/copy2clipboard-button.directive.ts

```diff
@@ -8,6 +8,8 @@ import { ToastrService } from 'ngx-toastr';
 export class Copy2ClipboardButtonDirective implements OnInit {
   @Input()
   private cdCopy2ClipboardButton: string;
+  @Input()
+  logFlag: boolean;
```

Comment:
This is a too ad-hoc variable name... What do we really want here? To extend this directive to either capture some HTML element value by ID or an Angular component property, right? What about something like `byId = true` and only when set to false we directly look for a component property?

Reply:
Done.